frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -104,6 +104,13 @@ const talentRoutes = (
       redirect: talentPaths.search(),
     }),
   },
+  {
+    path: `${talentPaths.home()}/search`,
+    action: () => ({
+      component: <div />,
+      redirect: talentPaths.search(),
+    }),
+  },
```

Comment:
This looks like a duplicate. There's a route for the search page above.

Reply:
Nice catch, that has been removed in this typo'd commit [remove duplicat search route](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3665/commits/452acfaa4af5c72d72ec6aa97d7716b164d2b162) 😆